go-controller/pkg/cni/cni_dpu.go

```diff
@@ -4,9 +4,35 @@ import (
 	"fmt"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+
+	"k8s.io/client-go/util/retry"
 )
 
+// updatePodDPUConnDetailsWithRetry update the pod annotion with the givin connection details
+func (pr *PodRequest) updatePodDPUConnDetailsWithRetry(kube kube.Interface, dpuConnDetails *util.DPUConnectionDetails) error {
+	resultErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		// Informer cache should not be mutated, so get a copy of the object
+		pod, err := kube.GetPod(pr.PodNamespace, pr.PodName)
```

Comment:
use the pod informer

Reply:
this call is from cni, do we also use informer instead of kube Api?

Reply:
we have a podLister in the cniserver https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/cni/cniserver.go#L73